src/test/java/seedu/address/logic/commands/task/DeleteTaskCommandTest.java

```diff
@@ -119,7 +119,7 @@ public void execute_taskRemovedFromMultiplePersons() throws Exception {
     }
 
     @Test
-    public void execute_taskRemoveIndexFailure() throws Exception {
+    public void execute_taskRemoveIndexFailure() {
```

Comment:
Sorry I'm not too sure why this is in this PR but this change is due to the function not throwing any exceptions